docs/semgrep-ci/network-broker.md

```diff
@@ -125,6 +125,34 @@ You can check the logs for Semgrep Network Broker by running:
 
 <pre class="language-console"><code>sudo docker logs <span className="placeholder">CONTAINER_ID</span></code></pre>
 
+### Adjusting log verbosity
+
+The Semgrep Network broker can log details of the proxied requests and responses for troubleshooting. Normally, these should only be collected when working to identify an issue. If requests and responses are large, they can occupy significant memory in the tunnel.
+
+In your broker configuration, that looks like:
```

Comment:
Maybe:

> To log additional details, add this snippet to your broker configuration:

Reply:
Valid, but I don't know how well it works with the warning above 🤔 
I'd prefer to merge it as is, but I don't feel strongly - if you prefer this, I'm happy to accept it as a suggestion / push and then merge.

Reply:
Yeap, The specific bit I was thinking about was "that looks like" - it seemed ambiguous. There's no indication of "if i want to troubleshoot, I should copy this thing" - if you are troubleshooting something, additional logging is a valid step to take even if it doesn't always lead to the solution.

I would have suggested "To log additional details for troubleshooting, add this snippet..." However, the text says "normally" used for troubleshooting - so there are other cases we may not know why a user wants to add logs. I will make the suggestion, thanks for being open to it.

Reply:
```suggestion
To log additional details, add this snippet to your broker configuration:
```